cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -365,6 +367,7 @@ The following commands are only executable against the `system` database:
 | <<administration-security-administration-dbms-privileges-user-management, SET PASSWORDS>>               | DBMS                  | Determines whether the user can modify users' passwords and whether those passwords must be changed upon first login.
 | <<administration-security-writes-set-property, SET PROPERTY>>                                           | GRAPH                 | Determines whether the user can set a property to an element (node, relationship or both) using the SET clause.
 | <<administration-security-administration-dbms-privileges-user-management, SET USER STATUS>>             | DBMS                  | Determines whether the user can modify the account status of users.
+| <<administration-security-administration-dbms-privileges-user-management, SET USER NAME>>               | DBMS                  | Determines whether the user can modify the name of users.
```

Comment:
I think you need to add set default database also? I think that is a privilege as well at least